src/app/project-overview/members-overview/template.html

```diff
@@ -20,9 +20,7 @@
       <div fxLayout="row"
            fxLayoutAlign=" center"
            fxLayoutGap="10px">
-        <span>{{showMembers ? 'Owners & Members' : 'Owners'}}</span>
-        <a class="members-link"
-           [routerLink]="['../members']"><i class="km-icon-next km-icon-mask km-no-margin"></i></a>
+        <span>{{showMembersAndGroups ? 'Owners, Members & Groups' : 'Access'}}</span>
```

Comment:
```suggestion
        <span>{{showMembersAndGroups ? 'Owners, Members & Groups' : 'Owners'}}</span>
```

Reply:
It's been decided to change the headline to "**Access**" for everyone, the icons can get tooltips.